driver/src/models/standing_order.rs

```diff
@@ -17,6 +21,24 @@ impl StandingOrder {
     }
 }
 
+impl RollingHashable for Vec<StandingOrder> {
```

Comment:
Rolling Hashable is already implemented for any Vector of type T as long as this type T implements `Serializable` (https://github.com/gnosis/dex-services/blob/master/driver/src/models/order.rs#L47). Therefore, instead of implementing `rolling_hash` again, all you need to do is implement Serializable for StandingOrder.

To deal with potentially missing accounts, I'd just create "empty" StandingOrder structs in the Vector for them. This way you can reuse the vector when you try to extract all the batch indexes later (instead of rewriting this relatively complex `.iter.position.map.map` again).